packages/plugin-less/src/index.ts

```diff
@@ -173,7 +168,15 @@ export const pluginLess = (
         callback: (rule: RspackChain.Rule, type: 'normal' | 'inline') => void,
       ) => {
         callback(rule, 'normal');
-        callback(inlineRule, 'inline');
+
+        // Rsbuild < 1.3.0 does not have RULE.CSS_INLINE.
+        if (chain.module.rules.has(CHAIN_ID.RULE.CSS_INLINE)) {
+          const inlineRule = chain.module
+            .rule(findRuleId(chain, CHAIN_ID.RULE.LESS_INLINE))
```

Comment:
findRuleId should not be put inside the updateRules method. It should be only called once.